### DISCUSSION TITLE
Tables not loading in dev mode

### DISCUSSION THREAD

**Discussion by RajGM** [AntlerVC/firetable#942] [Support / Q&A] [answered]
I am unable to load any table in dev mode.
http://localhost:7699/tables
It's showing No tables
@notsidney @htuerker

**Comment by notsidney** [comment, marked as answer]
Please make sure that you have the required Firestore Rules set. https://docs.rowy.io/how-to/roles#write-firestore-security-rules-with-user-roles

Also make sure you have your account set with the `ADMIN` role using the instructions on that page.
